collections/_article/dev-snapshot-godot-4-6-beta-1.md

```diff
@@ -0,0 +1,399 @@
+
+And more:
+- Add methods to draw ellipses ([GH-85080](https://github.com/godotengine/godot/pull/85080)).
+- Apply viewport oversampling to Polygon2D ([GH-112352](https://github.com/godotengine/godot/pull/112352)).
+- Implement a very simple SSAO in GLES3 ([GH-109447](https://github.com/godotengine/godot/pull/109447)).
+- Implement motion vectors in compatibility renderer ([GH-97151](https://github.com/godotengine/godot/pull/97151)).
+- Overhaul and optimize Glow in the mobile renderer ([GH-110077](https://github.com/godotengine/godot/pull/110077)).
+- Rewrite Radiance and Reflection probes to use Octahedral maps ([GH-107902](https://github.com/godotengine/godot/pull/107902)).
+- TAA adjustment to reduce ghosting ([GH-112196](https://github.com/godotengine/godot/pull/112196)).
+
+### XR
+
+[Bastiaan Olij](https://github.com/BastiaanOlij) has brought native support for OpenXR 1.1 to Godot, which will automatically enable OpenXR 1.1 features on devices that support it ([GH-109302](https://github.com/godotengine/godot/pull/109302)). This comes with a compatibility layer to make this as transparent as possible for developers who want to deploy their games both to headsets that support OpenXR 1.1, and those that only support OpenXR 1.0.
+
+Bastiaan additionally implemented support for [OpenXR Spatial Entities Extensions](https://www.khronos.org/blog/openxr-spatial-entities-extensions-released-for-developer-feedback), introduced to standardize obtaining and interacting with information about the user's real world environment ([GH-107391](https://github.com/godotengine/godot/pull/107391)). This brings support for [spatial anchors](https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_EXT_spatial_anchor), [plane tracking](https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_EXT_spatial_plane_tracking), and [marker tracking](https://registry.khronos.org/OpenXR/specs/1.1/html/xrspec.html#XR_EXT_spatial_marker_tracking) in a consistent and platform-independent manner. If you have the necessary equipment for it, Bastiaan has provided a [demo project](https://github.com/BastiaanOlij/spatial-entities-demo) to showcase this new functionality.
+
+[dhoverb](https://github.com/dhoverb) took to supporting the extensions for offsetting density maps ([GH-112888](https://github.com/godotengine/godot/pull/112888)). This is a key feature enabling eye-tracked foveated rendering on devices that support this method. Currently, this has been tested on Meta Quest Pro and Samsung Galaxy XR headsets, but other headsets are expected to benefit from this change as well.
```

Comment:
If we're doing human names:

```suggestion
[David Hover](https://github.com/dhoverb) took to supporting the extensions for offsetting density maps ([GH-112888](https://github.com/godotengine/godot/pull/112888)). This is a key feature enabling eye-tracked foveated rendering on devices that support this method. Currently, this has been tested on Meta Quest Pro and Samsung Galaxy XR headsets, but other headsets are expected to benefit from this change as well.
```

Reply:
Oh my goodness, I merged JUST as you submitted these! We'll catch these in a followup